client/driver/executor/executor.go

```diff
@@ -334,6 +346,31 @@ func (e *UniversalExecutor) ShutDown() error {
 	return nil
 }
 
+func (e *UniversalExecutor) RegisterServices() error {
+	e.logger.Printf("executor: registering services")
```

Comment:
Even though these logs don't make it to the main log I would still treat them the same. It is nice for an operator to see the severity